types/factory-girl/index.d.ts

```diff
@@ -65,18 +83,27 @@ declare namespace factory {
         /**
          * Builds a new model instance that is persisted
          */
-        create<T>(name: string, attrs?: Attributes<Partial<T>>): Promise<T>;
+        create<T>(name: string, attrs?: Attributes<Partial<T>>, buildOptions?: BuildOptions): Promise<T>;
 
         /**
          * Builds an array of model instances that are persisted
          */
-        createMany<T>(name: string, num: number, attrs?: Attributes<Partial<T>>, buildOptions?: Options<T>): Promise<T[]>;
-        createMany<T>(name: string, attrs?: ReadonlyArray<Attributes<Partial<T>>>, buildOptions?: Options<T>): Promise<T[]>;
+        createMany<T>(
+            name: string,
+            num: number,
+            attrs?: MaybeReadonlyArray<Attributes<Partial<T>>>,
+            buildOptions?: MaybeReadonlyArray<BuildOptions>,
```

Comment:
A `Options<T>` option can't be specified here. You can specify these options in `Factory#define()` or `Factory#extend()`.

- https://github.com/simonexmachina/factory-girl/blob/f7ec9d5004021ff1334ab9440e9b5cc4fd1a18b0/src/Factory.js#L110